src/info/langs/mod.rs

```diff
@@ -31,23 +31,23 @@ fn get_language_distribution(languages: &tokei::Languages) -> Option<HashMap<Lan
     let mut language_distribution = HashMap::new();
 
     for (language_name, language) in languages.iter() {
-        let mut code = language.code;
+        let mut loc = language::loc(language_name, language);
 
         let has_children = !language.children.is_empty();
 
         if has_children {
             for reports in language.children.values() {
                 for stats in reports.iter().map(|r| r.stats.summarise()) {
-                    code += stats.code;
+                    loc += stats.code;
```

Comment:
`language::loc` should probably also applied here in case one of the children language happens to be Markdown.

To handle this case for example:

![image](https://user-images.githubusercontent.com/13710835/213878103-be585674-086e-4891-82fa-508230f0448e.png)

Reply:
That's a good point. I don't use Jupyter notebooks, so I thought Markdown was the "top level" code. Could that Markdown then contain its own code blocks? If so, perhaps this function should be recursive for deeply nested code blocks?

Reply:
Indeed, tokei's parser is indeed recursive, as stated in its [changelog](https://github.com/XAMPPRocky/tokei/blob/master/CHANGELOG.md)

![image](https://user-images.githubusercontent.com/13710835/213888189-359049a9-88a0-450e-9e20-a5fa69b17336.png)

Reply:
Deeply nested stats are now recursively counted with https://github.com/o2sh/onefetch/pull/937/commits/939e9e1f1b5f78cb5a136ccf8dcaf1b932e64d00